Source/WebCore/loader/DocumentLoader.cpp

```diff
@@ -1112,8 +1112,7 @@ void DocumentLoader::continueAfterContentPolicy(PolicyAction policy)
             InspectorInstrumentation::continueWithPolicyDownload(*frame, *mainResourceLoader->identifier(), *this, m_response);
 
         RefPtr document = frame->document();
```

Comment:
The `else if (document)` below can be changed to `else if (RefPtr document = frame->document())` below, and this variable removed

Reply:
done. thanks.